nservicebus/security/encryption.md

```diff
@@ -47,8 +52,15 @@ Property encryption is enabled via the configuration API.
 
 <!-- import EncryptionServiceSimple -->
 
+### Generating encryption keys
+
+In order to use the RijndaelEncryptionService a primary and secondary keys need to be generated. An option is to use `RijndaelManaged.GenerateKey`, please refer to the following article to generate new keys:
+
+- https://msdn.microsoft.com/en-us/library/system.security.cryptography.rijndaelmanaged.generatekey(v=vs.110).aspx
```

Comment:
@DavidBoike Agreed, the primary/secondary line needs to be rewritten. What I wanted to say that all keys should to be generated using a secure pseudorandom generator.

@SimonCropp Agree, actual code is needed.